app/_src/gateway-operator/reference/version-compatibility.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Version Compatibility
+---
+
+## {{ site.kic_product_name }}
+
+The following table presents the general compatibility of {{site.kgo_product_name}} with {{ site.kic_product_name }} minor versions.
+
+## Kubernetes
+
+| {{site.kgo_product_name}}        |            1.0.x            |
+|:---------------------------------|:---------------------------:|
+| {{ site.kic_product_name }} 2.11 | <i class="fa fa-check"></i> |
+| {{ site.kic_product_name }} 2.12 | <i class="fa fa-check"></i> |
```

Comment:
```suggestion
| {{ site.kic_product_name }} 3.0 | <i class="fa fa-check"></i> |
```

Is this accurate?

Reply:
We have [just recently bumped the default `ControlPlane` version to 3.0](https://github.com/Kong/gateway-operator/pull/1189) which made KGO CI test with 3.0 ( we haven't been testing that before as 3.0 is quite a fresh topic ).

Having said that I've tested operator 1.0.3 with KIC 3.0 and one thing that doesn't work is the `ClusterRole` that operator uses for KIC (in particular version) doesn't work for KIC 3.0

Specifically: this one https://github.com/Kong/gateway-operator/blob/v1.0.3/internal/utils/kubernetes/resources/clusterroles/zz_generated_controlplane_clusterrole_ge2_12.go is for versions >= 2.12 but doesn't include `KongUpstreamPolicy` rules which makes KIC to use a role which doesn't allow it to watch/list resources of that type:

```
W1116 09:46:24.884067       1 reflector.go:535] pkg/mod/k8s.io/client-go@v0.28.3/tools/cache/reflector.go:229: failed to list *v1beta1.KongUpstreamPolicy: kongupstreampolicies.configuration.konghq.com is forbidden: User "system:serviceaccount:default:controlplane-kong-gl72n-wgs5b" ca
nnot list resource "kongupstreampolicies" in API group "configuration.konghq.com" at the cluster scope
E1116 09:46:24.884113       1 reflector.go:147] pkg/mod/k8s.io/client-go@v0.28.3/tools/cache/reflector.go:229: Failed to watch *v1beta1.KongUpstreamPolicy: failed to list *v1beta1.KongUpstreamPolicy: kongupstreampolicies.configuration.konghq.com is forbidden: User "system:serviceacco
unt:default:controlplane-kong-gl72n-wgs5b" cannot list resource "kongupstreampolicies" in API group "configuration.konghq.com" at the cluster scope
```

This causes KIC Pods to go in a perpetual restart loop.

--- 

I have created https://github.com/Kong/gateway-operator/issues/1200 to discuss how we'd like to proceed with this one but as of now: KGO 1.0.3 does not support KIC 3.0 due to the missing `ClusterRole` rules.

Reply:
```suggestion
| {{ site.kic_product_name }} 3.0  | <i class="fa fa-times"></i> |
```

Reply:
In the meantime, [KGO v1.1.0](https://github.com/Kong/gateway-operator/issues/1205) will introduce support for KIC 3.0+ via https://github.com/Kong/gateway-operator/pull/1189